python/desc/imsim/imSim.py

```diff
@@ -178,3 +202,32 @@ def extract_objects(df):
     phosim_galaxies = \
         phosim_galaxies.query("halfLightSemiMajor >= halfLightSemiMinor")
```

Comment:
That's right, it was on linux where it didn't crash.  Using the old code, I did run into that message complaining about a single galaxy at the 1e-22 level, but with the new code, `query("halfLightSemiMajor >= halfLightSemiMinor")` has no effect, i.e., all rows are passed and the code runs fine.  So we can add a check, but it wouldn't do anything on linux.   What other validations would you suggest?